test/unit/slurm/test_slurm_config_generator/test_generate_slurm_config_with_custom_settings/sample_input.yaml

```diff
@@ -0,0 +1,86 @@
+Scheduling:
```

Comment:
I would propose to make this file as minimal as possible in order to be able to test the rendering of the custom Slurm settings. Let's just define the queue names, the compute resources names and the custom Slurm settings (plus any parameter absolutely necessary for the rendering of the include files to succeed).

Reply:
I was talking about the test input yaml, not about any logic in the Python code.